packages/@okta/vuepress-site/docs/guides/terraform-manage-multiple-domains/main/index.md

```diff
@@ -0,0 +1,262 @@
+
+For guidance on organizing your files, see [setting up a typical Okta Terraform configuration](/docs/guides/terraform-organize-configuration/main/#configure-a-basic-okta-terraform-configuration). Consider organizing your Terraform code in a way that groups related resources together. For example, you could create a Terraform file called `brands.tf` that contains custom domains, brands, and themes.
+
+## Add or confirm the API scopes
+
+To manage domains, the API scopes in your Terraform integration must include `okta.domains.manage`.
+
+To grant scopes in the Admin Console and to include them in your Terraform code, see the articles on [enabling your API service application for Terraform access](/docs/guides/terraform-enable-org-access/) and [setting up a typical Okta Terraform configuration](/docs/guides/terraform-organize-configuration/).
+
+## Add a custom domain in Terraform
+
+Creating and adding a custom domain takes several steps that must be completed in order. At least one of the steps may require waiting for up to a day. The steps are:
+
+1. [Create an okta-domain resource](#create-a-custom-domain-resource).
+
+1. [Update the domain with the `CNAME` and `TXT` records](#update-the-domain-with-the-cname-and-txt-records).
+
+1. [Request domain verification](#request-domain-verification).
+
+1. [Confirm your custom domain triggers Okta sign-in](#confirm-your-custom-domain-triggers-okta-sign-in).
+
+1. [Update other objects and systems to use your custom domain](#update-other-objects-and-systems-to-use-your-custom-domain).
+
+### Create a custom domain resource
+
+1. Add an [okta-domain](https://registry.terraform.io/providers/okta/okta/latest/docs/resources/domain) Terraform resource to your configuration for the new domain.
+
+1. Set the `name` field to the domain name, such as `"example.com"`.
+
+1. Set the certificate source type:
+
+    It's recommended to enable Okta management of the certificate by setting the source-type attribute:
+
+      `certificate_source_type = "OKTA_MANAGED"`
+
+    Some organizations and pricing options do not support certificates managed by Okta. If you are unsure or want to migrate a manual certificate to an Okta-managed certificate, contact Okta Support.
+
+    The following code shows an example resource for an Okta-managed certificate for the domain `company1.example.com` of company1.:
+
+    ```hcl
+    resource "okta_domain" "company1" {
+      name = "company1.letsauth0.com"
+      certificate_source_type = "OKTA_MANAGED"
+    }
+    ```
+
+> **Note:** If you maintain the custom domain's [Certificate Authority Authorization (CAA)](https://datatracker.ietf.org/doc/html/rfc6844) record, add `letsencrypt.org` to the issuers list. This enables Okta to obtain and renew the TLS certificate. You must also do this if you start maintaining the CAA record after first only using the Okta certificate. For more information, see [Let's Encrypt—Using CAA](https://letsencrypt.org/docs/caa/).
```

Comment:
```suggestion
> **Note:** If you maintain the custom domain's [Certificate Authority Authorization (CAA)](https://datatracker.ietf.org/doc/html/rfc6844) record, add `letsencrypt.org` to the issuers list. This enables Okta to obtain and renew the TLS certificate. You must also do this if you initially used only the Okta certification, but are now maintaining the CAA record. See [Let's Encrypt—Using CAA](https://letsencrypt.org/docs/caa/).
```

Reply:
A slight change: the portion above that reads "Okta certification, but" should read "Okta certificate, but"... which is how the change is implemented in the next commit.